dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/oracle/V6.0_2015.03.07__DS-2701_Hibernate_migration.sql

```diff
@@ -343,14 +384,17 @@ UPDATE metadatavalue SET place = -1 WHERE place IS NULL;
 -- Alter harvested item
 ALTER TABLE harvested_item RENAME COLUMN item_id to item_legacy_id;
 ALTER TABLE harvested_item ADD item_id RAW(16) REFERENCES item(uuid);
+CREATE INDEX harvested_item_item on harvested_item(item_id);
 UPDATE harvested_item SET item_id = (SELECT item.uuid FROM item WHERE harvested_item.item_legacy_id = item.item_id);
 ALTER TABLE harvested_item DROP COLUMN item_legacy_id;
 
 -- Alter harvested collection
 ALTER TABLE harvested_collection RENAME COLUMN collection_id to collection_legacy_id;
 ALTER TABLE harvested_collection ADD collection_id RAW(16) REFERENCES Collection(uuid);
+CREATE INDEX harvested_collection_collectio on harvested_collection(collection_id);
```

Comment:
This had to be truncated because Oracle won't accept identifiers longer than 30 characters.  There are several other truncations.